### PR TITLE
Portfolio: finish mobile version

### PR DESCRIPTION
**Pull request by Ferizad**
These sections of the mobile website should include:
"Works" section
the "About myself" section
Use ul and li for categories in "Work" section. Html, Css, and Javascript should be list items.
Use ul and li for categories in "About myself" section. Languages, Frameworks, and Skills should be list items.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,296 @@
...
+  margin: 0;
+  padding-bottom: 40px;
+}
+
+.cards {
+  display: grid;
+  justify-content: center;
+  padding: 10px 10px 10px 10px;
+  grid-template-columns: 330px;
+}
```
Comment (emmiiorji) [inline style.css]: Well done for following the figma design quite closely. It is [required](https://github.com/microverseinc/curriculum-html-css/blob/e1fd98988a32ecaf3767a9832e6cea0ff0d18792/portfolio/2_mobile.md?plain=1#L51) to have the container holding all projects have a grid layout while the individual projects have a display: flex. You have the container for all projects to be `.works` while each project is enclosed in `.cards`

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi @Ferizad, 
# Status: Changes Required ♻️

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

<div align='center'>

</div>

## Good Highlights✔️

- Linter checks are passing
- Good commit messages
- Proper use of GitHub flow
- Good design


## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (Ferizad) [reply style.css]: @emmiiorji thanks for your feedback. But it seems like I confused you by giving different names to my classes. So the main container class in my project here is ".cards" and cards itself is a ".card" class. 
I used the grid  ".cards" class and flex in ".card" class respectively as per the requirement you mentioned above.

**Review by emmiiorji — APPROVED** [review]
Hi @Ferizad, 

# Status: Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


<div align='center'>

</div>


## Good Highlights✔️

- Linter checks are passing
- Good commit messages
- Proper use GitHub flow
- Good design

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (emmiiorji) [reply style.css]: Hi @Ferizad,

I see, you're quite correct. I have approved accordingly.

**Pull request merged**